vmm/src/config.rs

```diff
@@ -2157,12 +2187,77 @@ impl RestoreConfig {
             .map_err(Error::ParseRestore)?
             .unwrap_or(Toggle(false))
             .0;
+        let net_ids = parser
```

Comment:
This should be pairs of values - look at `DiskConfig::queue_affinity` for an example.

Reply:
Implemented the new approach with `RestoredNetConfig`